### ISSUE TITLE
Changes over in Query.jl land

### ISSUE THREAD

**Issue by davidanthoff** [bramtayl/LazyQuery.jl#2]
Just a heads up that I'm changing some things over in Query.jl that might affects this package.

I'm factoring out the basic query operators and the enumerable backend implementation into https://github.com/davidanthoff/QueryOperators.jl. Going forward, Query.jl will only have the front-end stuff, i.e. the user-facing macros.

I think the ideal situation would be if you could just rely on QueryOperators.jl, and then on the stuff in https://github.com/davidanthoff/QueryOperators.jl/blob/master/src/operators.jl. But I'm not sure that would work for you? In any case, that is kind of the "public" interface that I plan to keep stable/supported.

Happy to help sort this out, let me know if things are unclear!

**Comment by bramtayl** [comment]
Yup sounds good! I'll try to make updates this weekend. Shouldn't be too hard, I don't think. PS I might use this thread for misc Query questions (Query queries) if thats ok

**Comment by davidanthoff** [comment]
Yes, definitely!

**Comment by bramtayl** [comment]
So I probably won't get to this after finals. I'm not entirely sure how useful LazyQuery is though. It hasn't gotten a lot of use in comparison to Query and the new standalone macros are close enough to dplyr that something even closer might not be worth it.

**Comment by davidanthoff** [comment]
I have to say some of the ideas in LazyQuery are really nice and I'm still mulling how one could bring some of those ideas over to Query.jl... It is one of those slow projects, I keep thinking about it periodically, and hopefully at some point there will be some good idea ;)
